api_app/analyzers_manager/models.py

```diff
@@ -0,0 +1,87 @@
+from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+from cache_memoize import cache_memoize
+
+from intel_owl.secrets import get_secret
+from api_app.models import Job
+
+
+class Analyzer(models.Model):
+    analyzer_type = models.CharField(
+        max_length=50,
+        choices=(
```

Comment:
can you extract out the `choices` into a constant variable inside the `Analyzer` model ? thanks